### ISSUE TITLE
Reduce history data size for future saving to JSON

### ISSUE THREAD

**Issue by a19hammi** [HGustavs/LenaSYS#10593]
Only relevant changes should be stored in the change in elements.
For example creating an element will write to the history:
```js
[
  {
    "id": "ABC123",
    "name": "Element",
    "x": 60,
    "y": 60,
    "w": 120,
    "h": 60
  }
]
```

Then after a certain time when the change would be considered DIFFERENT a new change in movement X of 10 coordinate units only would result in the history being:
```js
[
  {
    "id": "ABC123",
    "name": "Element",
    "x": 60,
    "y": 60,
    "w": 120,
    "h": 60
  },
  {
    "id": "ABC123",
    "x": 10
  }
]
```

We finally need a form of indicator for deletion of an element, how you solve it without much data usage that would be really nice! :)

**Comment by HGustavs** [comment]
My idea/pseudocode for storing deletion is available in issue #10304

**Comment by a19hammi** [comment]
fixed in #10702
